### ISSUE TITLE
API call /system/cluster/stats fails with 500

### ISSUE THREAD

**Issue by fungusakafungus** [Graylog2/graylog2-server#1058]
```
curl http://graylog2.staging:12900/system/cluster/stats
{"type":"ApiError","message":""}
```

server log shows:

```
2015-03-12T10:58:07.178Z ERROR [AnyExceptionClassMapper] Unhandled exception in REST resource
java.lang.NullPointerException
        at org.graylog2.system.stats.mongo.MongoProbe.mongoStats(MongoProbe.java:132)
        at org.graylog2.system.stats.ClusterStatsService.mongoStats(ClusterStatsService.java:92)
        at org.graylog2.system.stats.ClusterStatsService.clusterStats(ClusterStatsService.java:68)
        at org.graylog2.rest.resources.system.ClusterStatsResource.systemStats(ClusterStatsResource.java:50)

```

Looks like MongoProbe.java tries to use dbStats.extentFreeList, which is new in mongodb version 3.0.0.

We use mongodb 2.0.6 and graylog-server 1.0.0

**Comment by joschi** [comment]
This has been fixed in the `1.0` branch (#1032) and will be shipped with Graylog 1.0.1.

> We use mongodb 2.0.6 and graylog-server 1.0.0

I guess you meant to write MongoDB 2.6, because Graylog requires MongoDB 2.2.x as a minimum supported version.

**Comment by fungusakafungus** [comment]
Thanks!
